### PR TITLE
Update teams-for-vdi.md

### PR DESCRIPTION
**Pull request by sadafbadar**

### CODE REVIEW

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -483,6 +483,10 @@ if($cleanup){
 - Azure Virtual Desktop doesn't support macOS and Linux-based clients at this time.
 - Fast tenant switch can result in calling-related issues on VDI such as screen sharing not available. Restarting the client will mitigate these issues.
 
+### Notifications
+
+- Message count notification/presence on Windows task bar is not supported on Windows Server 2016 host.
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md:488]:
```suggestion
- Message count notification and presence on the Windows taskbar is not supported on Windows Server 2016 host.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sadafbadar, kindly commit the suggested changes. Many thanks.

**Comment by sadafbadar** [comment]
@JohanFreelancer9 Changes looks good. Please publish it!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**